src/components/SongList/SongList.js

```diff
@@ -4,12 +4,12 @@ import PropTypes from 'prop-types'
 import React from 'react'
 
 const SongList = ({
-  changeVote,
-  isEditing,
-  songs,
-  toggleIsEditing,
-}) => (
-  <div className='SongList'>
+                    changeVote,
+                    isEditing,
+                    songs,
+                    toggleIsEditing,
+                  }) => (
```

Comment:
is this indentation a webstorm thing?

Reply:
Yeah, not sure why it does that. I have my autoformatter set up with standardJS and its pretty picky about spacing and it just does stuff like that on save